packages/components/text/src/text.tsx

```diff
@@ -1,3 +1,4 @@
+// TODO: @redesign cleanup
```

Comment:
We can remove this comment now since we're not code branching for the theme migration in this file anymore.

Reply:
Thank you.
Updated here: https://github.com/commercetools/ui-kit/pull/2370/commits/1e63f655500515fd884221f1b9e837b9d5ad8260